guides/common/modules/proc_enable-insights-with-capsule.adoc

```diff
@@ -0,0 +1,31 @@
+[id='register-capsule-with-insights_{context}']
+
+<<<<<<< HEAD
+= Register :SmartProxyServer: with Red Hat Insights
+=======
+= Register a Capsule server with Red Hat Insights
+>>>>>>> eae01c40c0df1d7b50c7736e22a47ae74f2e5fc3
```

Comment:
I noticed you have a bunch of conflict related git statements in here that needs cleaning